test/fixtures/open_with_integrations/multipleIntegrations.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
I think you'll want the file/folder to be named something like `open-with/integrations-multiple.json`.

Reply:
agreed, it's not worth trying to keep one folder per endpoint, replicating them isn't a huge deal.